src/components/login/index.js

```diff
@@ -7,11 +7,16 @@ module.exports = (app) => {
     */
     const Login = {
         computed: app.helpers.sharedComputed(),
+        created: function() {
```

Comment:
Just use Vue-stash store: {} to mount the store property in the component, instead of copying it to data.